src/app.py

```diff
@@ -8,11 +9,17 @@
 app.config["SECRET_KEY"] = "rachel27"
 
 # reading in data for characters and spells
-hp_characters = "../data/Harry_Potter_Characters.tsv"
+hp_characters = (
+    "D:/Duke/Lectures/07_BIO_821_PYTHON_TOOLS/"
+    "biostats_finalproject_hp/data/Harry_Potter_Characters.tsv"
+)
```

Comment:
Just a note to change this back to reading from the data folder before finishing